plugins/shared-react/src/types/pipeline.ts

```diff
@@ -0,0 +1,137 @@
+import { PipelineRunKind } from './pipelineRun';
```

Comment:
should put these (similar files) under dir pipelines if this is for pipelines specific which can be used in topology and tekton